glustercli/cmd/common.go

```diff
@@ -20,6 +21,8 @@ var (
 
 func initRESTClient(hostname, user, secret, cacert string, insecure bool) {
 	client = restclient.New(hostname, user, secret, cacert, insecure)
+	client.ExtendAgent(
+		fmt.Sprintf("glustercli/%v", version.GlusterdVersion))
```

Comment:
nit: We can have cli version instead of glusterd version. For now it is same, but it can change in future. @prashanthpai  your thoughts?

Reply:
IMO, this should be okay. glustercli and glusterd are almost always built together. We can quickly identify if older/outdated clients connect to more recent glusterd2 by looking at the version if they follow same versioning. A separate version for CLI will makes more sense if it's a separate repo/project.

Reply:
makes sense. we can update later if that happens.